sktime/data/util.py

```diff
@@ -3,7 +3,10 @@
 
 def timeshifted_split(inputs, lagtime: int, chunksize=None, n_splits=None):
     assert lagtime > 0
-    assert (chunksize is not None) ^ (n_splits is not None)
+    #assert (chunksize is not None) ^ (n_splits is not None)
+    if chunksize is None and n_splits is None:
+        chunksize = 500
```

Comment:
this magic number is going to bite us at some point

Reply:
I replaced it with a default argument. Thanks good catch.